Documentation/tutorial/intro_template.md

```diff
@@ -0,0 +1,170 @@
+Introduction to *DocFX Template System*
+===============================
+
+*DocFX Template System* provides a flexible way of defining and using templates.
+
+As the following DocFX workflow shows,
+
+![DocFX workflow](../spec/images/docfx_workflow_highlevel.png)
+
```

Comment:
I think the figure here is to introduce terms. Actually the explanation itself introduces in new terms, and makes it even more difficult to understand. I'd prefer leaving the explanation to the below sections.

---
In reply to: [63306612](https://github.com/dotnet/docfx/pull/306#discussion_r63306612) [](ancestors = 63306612)